docs/config.rst

```diff
@@ -40,6 +40,15 @@ updating your stacks. By default it uses a bucket named
 If you want to change this, provide the **stacker_bucket** top level key word
 in the config.
 
+Module Paths
+----------------
+When setting the ``classpath`` for blueprints/hooks, it is sometimes desirable to
+load modules from outside the default ```sys.path`` (e.g., to include modules
```

Comment:
@phobologic good catch, should be fixed now.

rst is new to me, so I was using the [syntax from their doc site](http://docutils.sourceforge.net/docs/user/rst/quickref.html#inline-markup)